src/types.ts

```diff
@@ -2,6 +2,7 @@ export type SignatureAlgorithm = "sha1" | "sha256" | "sha512";
 
 export interface SamlSigningOptions {
   privateKey: string | Buffer;
+  signingCert?: string | string[] | null;
```

Comment:
We don't use this pattern very much anymore. It makes the code cleaner if we don't have to correct the type and do type narrowing. We'd rather the user just pass the correct type. If array is supported, just make this `string[] | null`. However, it doesn't seem, from the code below, that an array actually brings any value.

Reply:
Actually, this is the same certificate used in "generateServiceProviderMetadata()", so I followed same convention used in service provider metadata.
![image](https://user-images.githubusercontent.com/6399781/162631643-16d8fb50-deda-4c93-9749-8a9c698d4464.png)

Reply:
I thought that this might be the same. You'll note that in this function, the array actually brings value, in the function you've added, the array does nothing. I think some consideration might need to be given to how to unite these two concepts.